### ISSUE TITLE
Does the "Internal" version of the Redstone Transmitter/Receiver have any applicaion?

### ISSUE THREAD

**Issue by Antifluxfield** [Blood-Asp/GT5-Unofficial#1068]
Seems like it takes/sends signals directly into the block itself... But do they have any practical usage?

**Comment by OvermindDL1** [comment]
Turn on or off a machine for example, open and close a grate that is also on the same block/pipe, etc... etc...

**Comment by Antifluxfield** [comment]
@OvermindDL1 
But the Machine Controller cover cannot be triggered by that "Internal" signal.
An external signal provided by the Redstone Receiver (Out) on the adjacent block is able to turn on the machine, but I failed with that Redstone Receiver (In), which I tried both on the adjacent block and on the machine itself.
Edit:
After searching through the codes for a while, I just find out that the Microwave Energy Transmitter and the Teleporter can response to the "Internal" receiver signal; but besides that, these two covers seems useless...
Or do I ignored something?
